cmd/zc_filter.go

```diff
@@ -225,3 +227,75 @@ func (fs filterSet) GetEnumerationPreFilter(recursive bool) string {
 	}
 	return prefix
 }
+
+////////
+
+// includeAfterDateFilter includes files with Last Modified Times >= the specified threshold
+// Used for copy, but doesn't make conceptual sense for sync
+type includeAfterDateFilter struct {
+	threshold time.Time
+}
+
+func (f *includeAfterDateFilter) doesSupportThisOS() (msg string, supported bool) {
+	msg = ""
+	supported = true
+	return
+}
+
+func (f *includeAfterDateFilter) appliesOnlyToFiles() bool {
+	return true
+	// because we don't currently (May 2020) have meaningful LMTs for folders. The meaningful time for a folder is the "change time" not the "last write time", and the change time can only be obtained via NtGetFileInformation, which we don't yet call.
+	// TODO: the consequence of this is that folder properties and folder acls can't be moved when using this filter.
+	//       Can we live with that, for now?
```

Comment:
sure, maybe this could be communicated in the flag's description?

Reply:
Sure. Will do.